rpcs3/Emu/Cell/lv2/sys_fs.cpp

```diff
@@ -1928,29 +1928,18 @@ error_code sys_fs_fcntl(ppu_thread& ppu, u32 fd, u32 op, vm::ptr<void> _arg, u32
 
 		const cfg::device_info device = g_cfg_vfs.get_device(g_cfg_vfs.dev_usb, vpath);
 
-		if (device.path.empty())
+		if (device.path.empty() || device.vid.empty() || device.pid.empty())
 		{
+			sys_fs.error("sys_fs_fcntl(0xc0000015): invalid device (%s)", vpath);
```

Comment:
This is a useless error, it provides nothing in terms of what's wrong.

Reply:
Hi, I've updated the error message.

Reply:
No it's not a real error, just test any game with it and you would it spamming like hell.

Reply:
How about degrading the message to "warning" or just eliminate it as if nothing happened?

Reply:
Just remove it